### PR TITLE
Update send route

### PR DESCRIPTION
**Pull request by fBravoBravo**
Make the send email to subscriber and send email to all subscribers just one function that accepts and array as body

### CODE REVIEW

File: internal/app/newsletter/api/controllers/newsletter.go [hunk 1]
```diff
@@ -96,50 +96,46 @@ func DeleteSubscriber(c *fiber.Ctx) error {
...
-	return nil
-}
-
-func SendEmailToSubscriber(c *fiber.Ctx) error {
-	id := c.Params("id")
-
-	if id == "" {
+	if len(arrayOfUsers) == 0 {
 		return c.Status(fiber.StatusBadRequest).JSON(fiber.Map{
 			"error": "id is required",
```
Comment (n4xo-dev) [inline internal/app/newsletter/api/controllers/newsletter.go:109]: Inadequate error message

File: internal/app/newsletter/api/routers/routes.go [hunk 2]
```diff
@@ -27,5 +27,5 @@ func Setup(app *fiber.App) {
 	// subs.Get("/", controllers.ListSubscribers)
 	// subs.Get("/:id", controllers.GetSubscriber)
 	subs.Post("/send", controllers.SendEmailToSubscribers)
-	subs.Post("/send/:id", controllers.SendEmailToSubscriber)
+	subs.Post("/send/:id", controllers.SendEmailToSubscribers)
```
Comment (n4xo-dev) [inline internal/app/newsletter/api/routers/routes.go:30]: `:id` is unnecessary and unused.

**Review by n4xo-dev — CHANGES_REQUESTED** [review]

File: internal/app/newsletter/api/controllers/newsletter.go (hunk 1 above)
Comment (fBravoBravo) [reply internal/app/newsletter/api/controllers/newsletter.go]: done

**Comment by betterAvoka24** [comment]
@fBravoBravo add swagger comments on top of function for documentation

**Comment by fBravoBravo** [comment]
swagger docs added

**Comment by fBravoBravo** [comment]
@n4xo-dev could you review the swagger docs? there are some parts that I don't know well, don't know if it's correct

**Review by n4xo-dev — APPROVED** [review]

**Pull request merged**